### PR TITLE
fix: update default error code to 500

### PR DESCRIPTION
**Pull request by utsabc**
## Update the default error code to 500

The intention of this PR is to address the default error code returned by transformer to server is retryable if not specifically set by developer. This helps to treat any system error by transformer as internal server error from server and prevent event loss

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "test:ci": "jest --detectOpenHandles --coverage --notify --expand --silent --watchAll=false --maxWorkers=2",
     "test:silent": "npm run test -- --silent",
     "test:benchmark": "node benchmark/index.js",
-    "test:integration": "jest -c jest.config.integration.js --detectOpenHandles",
+    "test:integration": "jest -c jest.config.integration.js --detectOpenHandles --silent",
```
Comment (saikumarrs) [inline package.json:38]: We have `test:integration:silent` now (right below this script).

File: src/v0/util/index.js [hunk 2]
```diff
@@ -1351,11 +1351,11 @@ const errorStatusCodeKeys = ['response.status', 'code', 'status'];
  * @param {Number} defaultStatusCode default status code that has to be set
  * @returns {Number}
  */
-const getErrorStatusCode = (error, defaultStatusCode = 400) => {
+const getErrorStatusCode = (error, defaultStatusCode = 500) => {
```
Comment (saikumarrs) [inline src/v0/util/index.js:1354]: Let's create a constant for the default status code and use it.

File: package.json (hunk 1 above)
Comment (utsabc) [reply package.json]: Fixed

File: src/v0/util/index.js (hunk 2 above)
Comment (utsabc) [reply src/v0/util/index.js:1354]: Will address this later once the other PR is merged

**Review by chandumlg — APPROVED** [review]

File: src/v0/util/index.js (hunk 2 above)
Comment (saikumarrs) [reply src/v0/util/index.js:1354]: It's hardly a two-minute job.
Nevertheless, please create a new task card or add this to the scope of any open tasks.
CC @krishna2020

**Pull request merged**